From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sun, 23 Apr 2017 15:54:58 +0200
Subject: Use Debian packaged fonts

Remove Google Fonts link used by upstream and instead
use local fonts. Except for "Roboto Slab", all are
available on Debian.

Forwarded: not-needed
Author: Stefan Breunig <stefan-debian@yrden.de>
---
 docs/_themes/sphinx_rtd_theme_violet/layout.html | 1 -
 1 file changed, 1 deletion(-)

--- a/docs/_themes/sphinx_rtd_theme_violet/layout.html
+++ b/docs/_themes/sphinx_rtd_theme_violet/layout.html
@@ -23,7 +23,6 @@
   {% endif %}
 
   {# CSS #}
-  <link href='https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>
 
   {# OPENSEARCH #}
   {% if not embedded %}
